compiler/luci/pass/src/CircleQuantizer.cpp

```diff
@@ -49,6 +49,148 @@ namespace
 using namespace luci;
 using LayerParam = luci::CircleQuantizer::Options::LayerParam;
 
+// This function updates user-given input_type to match with the input signature of graph
+// If user gives only one input_type, it will be expanded to the number of graph inputs
```

Comment:
This explains why canonicalization is introduced.

Previously, users only gave one input/output_type. For backward-compatibility, we still allow users to give one input/output_type. For models with multiple inputs/outputs, we expand the one input/output_type to match with the number of inputs/outputs of a graph. I'd call the process canonicalization.